src/app/dao/entity.ts

```diff
@@ -2,109 +2,117 @@
  * Module dependencies.
  */
 import * as Sequelize from 'sequelize'
-import { Entity as entityModel } from '../models/entity'
+import { Entity } from '../models/entity'
 import { Request, Response } from '../../../node_modules/@types/express/index'
 
-export const get = (req: Request, res: Response) => {
```

Comment:
i suggest you don't use arrow function for methods, you can use it only for short statement (map, forEach, event) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
Done.